src/site/content/en/blog/how-to-use-local-https/index.md

```diff
@@ -0,0 +1,317 @@
+
+    Then, run:
+
+    ```bash
+    mkcert localhost
+    ```
+
+    Or if you're using a custom hostname like `mysite.example`:
+
+    ```bash
+    mkcert mysite.example
+    ```
+
+    The command above does two things:
+
+    - Generates a certificate for the hostname you've specified
+    - Lets mkcert—that you've added as a local CA in Step 2—sign this certificate.
+
+    Now, your certificate is ready and signed by a certificate authority your browser trusts locally. You're almost done, but your server doesn't know about your certificate yet!
+
+1.  **Configure your server.**
+
+    You now need to tell your server to use HTTPS (since development servers tend to use HTTP by default) and to use the TLS certificate you've just created.
+
+    How to do this exactly depends on your server. A few examples:
+
+    **👩🏻‍💻 With node:**
+
+    `server.js` (replace `{path/to/certificate...}` and `{port}`):
+
+    ```javascript
+    const https = require('https');
+    const fs = require('fs');
+    const options = {
+      key: fs.readFileSync('{path/to/certificate-key-filename}.pem'),
+      cert: fs.readFileSync('{path/to/certificate-filename}.pem'),
+    };
+    https
+      .createServer(options, function (req, res) {
+        // server code
+      })
+      .listen({port});
+    ```
+
+    **👩🏻‍💻 With [http-server](https://www.npmjs.com/package/http-server):**
+
+    Start your server as follows (replace `{path/to/certificate...}`):
+
+    ```bash
+    http-server -S -C {path/to/certificate-filename}.pem -K {path/to/certificate-key-filename}.pem
+    ```
+
+    `-S` runs your server with HTTPS, while `-C` and `-K` set the **c**ertificate and **k**ey.
+
+    **👩🏻‍💻 With a React development server:**
+
+    Edit your `package.json` as follows, and replace `{path/to/certificate...}`:
+
+    ```json
+    "scripts": {
+    "start": "HTTPS=true SSL_CRT_FILE={path/to/certificate-filename}.pem SSL_KEY_KILE={path/to/certificate-key-filename}.pem react-scripts start"
+    ```
+
+    For example, if you've created a certificate for `localhost` that is located in your site's root directory as follows:
+
+    ```text
+    |-- my-react-app
+        |-- package.json
+        |-- localhost.pem
+        |-- localhost-key.pem
+        |--...
+    ```
+
+    Then your `start` script should like as follows:
+
+    ```json
+    "scripts": {
+        "start": "HTTPS=true SSL_CRT_FILE=localhost.pem SSL_KEY_KILE=localhost-key.pem react-scripts start"
+    ```
+
+    **👩🏻‍💻 Other examples:**
+
+    - [Angular development server](https://angular.io/cli/serve)
+    - [Python](https://blog.anvileight.com/posts/simple-python-http-server/)
+
+    **Note:** Servers may use a different port for HTTPS.
+
+1.  ✨ You're done!
+    Open `https://localhost` or `https://mysite.example` in your browser: you're running your site locally with HTTPS.
+    You won't see any browser warnings, because your browser trusts mkcert as a local certificate authority.
+
```

Comment:
```suggestion

1.  ✨ You're done!
    Open `https://localhost` or `https://mysite.example` in your browser: you're running your site locally with HTTPS.
    You won't see any browser warnings, because your browser trusts mkcert as a local certificate authority.

{% Aside %} 
Remember that different servers may use a different port for HTTPS.
{% endAside %}

```